packages/io.prometheus/PklProject

```diff
@@ -18,5 +18,5 @@
 amends "../basePklProject.pkl"
 
 package {
-  version = "1.1.3"
+  version = "2.0.0"
```

Comment:
Is this a breaking change?

Reply:
Yup:
* Removed `bearer_token` and `bearer_token_file` - I could not find reference to these anywhere in the prom docs and this usage is covered by `Authorization`
* `RemoteWriteConfig.headers` was previously of type `Labels?`, I've updated it to `Mapping<LabelNameFormat, String>` as [the corresponding struct field is `map[string]string`](https://github.com/prometheus/prometheus/blob/58963f7b2e0b0ec345e3672d2e048627a8961fb3/config/config.go#L1106)

Reply:
Ideally, we try to avoid bumping major versions if it can be avoided. Introducing a new major version means that prometheus 1.x can't be reconciled with a 2.x version within a project, which can cause headaches WRT the diamond dependency problem.

For the two addressed breaking changes:

* `bearer_token` and `bearer_token_file` exist in [prometheus 1.8](https://prometheus.io/docs/prometheus/1.8/configuration/configuration/). Looks like they got removed somewhere? But their docs don't mention this anywhere as a breaking change.
* It's actually safe to keep `headers: Labels?`. Prometheus will accept ints/booleans as strings; the YAML `someValue: 5` gets parsed into into `map[string]string{"someValue": "5"}` thanks to how gopkg.in/yaml.v2 works

Can you undo these, and make this a minor version bump instead?

Reply:
Looking at this more, I think `bearer_token`, `bearer_token_file` [still exist](https://github.com/prometheus/common/blob/8742f090d9fd2cea7e53a4fd58ae5fb6c6634336/config/http_config.go#L306-L311)

Reply:
Huh, not sure how I messed that up. I thought I cmd-F'd that file for `bearer_token` and everything 😖

I've put those properties back, marked deprecated to match the source struct.